src/main/java/com/palantir/gradle/revapi/RevapiReportTask.java

```diff
@@ -85,10 +84,6 @@ private Path templateWithDifferenceTemplateAndWriteToFile(String resourceName, S
     }
 
     private File junitOutput() {
-        Optional<String> circleReportsDir = Optional.ofNullable(System.getenv("CIRCLE_TEST_REPORTS"));
-        File reportsDir = circleReportsDir
-                .map(File::new)
-                .orElseGet(() -> getProject().getBuildDir());
-        return new File(reportsDir, "junit-reports/revapi/revapi-" + getProject().getName() + ".xml");
+        return new File(getProject().getBuildDir(), "junit-reports/revapi/revapi-" + getProject().getName() + ".xml");
```

Comment:
I think this PR + our current CircleCI templates actually causes a bit of a functionality regression because our command is just:

https://github.com/palantir/gradle-revapi/blob/develop/.circleci/config.yml#L72
```
command: mkdir -p ~/junit && find . -type f -regex ".*/build/.*TEST.*xml" -exec cp --parents {} ~/junit/ \;
```

Reply:
Either we need to change the circleci template, or I think we should undo this and maybe just write some gradle to intentionally mess up the environment of the `test` task to unset the CIRCLE_TEST_REPORTS dir.

Reply:
Ooof, I'll see if I can get the gradle environment var changing thing to work.